src/commcare_cloud/ansible/wipe_postgres.yml

```diff
@@ -0,0 +1,31 @@
+---
+
+- name: Delete all PostgreSQL databases
+  hosts: postgresql:pg_standby:!citusdb
+  vars:
+    postgresql_host: "{{ inventory_hostname }}"
+  tasks:
+    - name: Check wipe_environment_enabled has been set to True
+      assert:
+        that: "{{ wipe_environment_enabled|default(False) }} == True"
+        fail_msg: 'This playbook will delete all data. To continue, set
+          "wipe_environment_enabled: True" in public.yml. Take care to unset
+          "wipe_environment_enabled" when the environment setup is complete.'
+
+    - name: Get PostgreSQL defaults
+      include_vars: roles/postgresql_base/defaults/main.yml
+
+    - name: Drop PostgreSQL databases
+      become_user: postgres
+      vars:
+        ansible_ssh_pipelining: true
+      with_items: "{{ postgresql_dbs.all }}"
+      # Only delete the databases we created (standby DB is not created, and not deleted)
+      when: item.create and ((item.host == postgresql_host) or is_monolith|bool)
+      postgresql_db:
+        name: "{{ item.name }}"
+        state: absent
+        port: "{{ postgresql_port }}"
+        login_host: 127.0.0.1
```

Comment:
I'm guessing this should be variable, something like
https://github.com/dimagi/commcare-cloud/blob/04725527f60167847eb88f3c990e8b51e42c3d38/src/commcare_cloud/environment/schemas/postgresql.py#L162-L166

Reply:
I thought so too, and I had to iterate on this a lot before I managed to get it to work. `environment.groups['postgresql'][0]` is going to be the external IP address (the address of `eth0` or `enp0s1` or whatever). But using that address doesn't work, because it's not the address in `pg_hba.conf`. The address you need is hardcoded here:
https://github.com/dimagi/commcare-cloud/blob/587843fa1c8f3cd190b59183190f870f4af872c5/src/commcare_cloud/ansible/roles/postgresql_base/templates/pg_hba.conf.j2#L33

It wasn't intuitive to me, so I think I'll add a comment to help the next person who finds this.

Reply:
sounds good - just wanted to check that it wasn't an oversight

Reply:
:+1: ... well, I'm sure there is something here I am missing. When these DBs are created, these values are omitted:
https://github.com/dimagi/commcare-cloud/blob/bcc391b57986a92a9f768cf5bc800dac2c980228/src/commcare_cloud/ansible/roles/postgresql_base/tasks/set_up_dbs.yml#L94-L96
I think this is because the command is using sockets, but I could not get that to work. I gave up, and went with `127.0.0.1` instead.